firehose/firehose.go

```diff
@@ -207,9 +209,13 @@ func (output *OutputPlugin) AddRecord(record map[interface{}]interface{}, timeSt
 		}
 	}
 
-	output.records = append(output.records, &firehose.Record{
-		Data: data,
-	})
+    if output.simpleAggregation && len(output.records) > 0 && len(output.records[len(output.records)-1].Data) + newDataSize <= maximumRecordSize {
+        output.records[len(output.records)-1].Data = append(output.records[len(output.records)-1].Data, data...)
```

Comment:
where are you adding the newline??

Reply:
For this part, new line has been added in `processRecord` in this [line](https://github.com/aws/amazon-kinesis-firehose-for-fluent-bit/blob/mainline/firehose/firehose.go#L282) as it used to. 
And the current output is like: 
```
{"cpu0_p_cpu":0,"cpu0_p_system":0,"cpu0_p_user":0,"cpu1_p_cpu":0,"cpu1_p_system":0,"cpu1_p_user":0,"cpu2_p_cpu":0,"cpu2_p_system":0,"cpu2_p_user":0}
{"cpu0_p_cpu":0,"cpu0_p_system":0,"cpu0_p_user":0,"cpu1_p_cpu":0,"cpu1_p_system":0,"cpu1_p_user":0,"cpu2_p_cpu":0,"cpu2_p_system":0,"cpu2_p_user":0}
```
I wonder if this is correct output or a newline expected like this:
```
{"cpu0_p_cpu":0,"cpu0_p_system":0,"cpu0_p_user":0,"cpu1_p_cpu":0,"cpu1_p_system":0,"cpu1_p_user":0,"cpu2_p_cpu":0,"cpu2_p_system":0,"cpu2_p_user":0}

{"cpu0_p_cpu":0,"cpu0_p_system":0,"cpu0_p_user":0,"cpu1_p_cpu":0,"cpu1_p_system":0,"cpu1_p_user":0,"cpu2_p_cpu":0,"cpu2_p_system":0,"cpu2_p_user":0}
```

Reply:
I see, you're good. We don't want two newlines.